src/index.css

```diff
@@ -439,11 +434,11 @@ button:active {
   }
 }
 
-.Preview__Pixel {
-  /*float: left;*/
+.Preview__Canvas {
+  display: block;
   position: relative;
-  width: var(--pixel);
-  height: var(--pixel);
+  width: calc(var(--pixel) * 8);
```

Comment:
is it needed to set the width/height here because it is already set to 100% inline.

Reply:
No need for this indeed. 
position:relative is not needed either.

Reply:
I'm not sure what is the best way to fix canvas width and height in grid and large preview, especially height.